test/lint/streams.rb

```diff
@@ -3,7 +3,7 @@
 module Lint
   module Streams
     MIN_REDIS_VERSION = '4.9.0'
-    MIN_REDIS_VERSION_XAUTOCLAIM = '6.2.0'
+    MIN_REDIS_VERSION_6_2 = '6.2.0'
```

Comment:
Might as well use the literal string directly.

Reply:
Does this comment mean that you'd like me to inline or you didn't like the name I chose?

Reply:
Please inline it.

Reply:
Inlined and squashed in 2c0f0dbdb6f465e12655dd7af0b812790d207103.